smartgymapi/handlers/device.py

```diff
@@ -54,6 +58,23 @@ def checkin(self):
             activity.gym = get_gym_by_MAC_address(result['client_address'])
             response["status"] = "checked in"
 
+            # we give units: metric for the api to return celcius
+            r_params = {"q": activity.gym.city,
```

Comment:
dit moet echt heel ernstig ergens een eigen methode krijgen. Dit is de derde keer dat we dit copy pasten